dev/openchpl_data-model.sql

```diff
@@ -2011,6 +2011,7 @@ CREATE TABLE openchpl.surveillance_nonconformity (
 	certification_criterion_id bigint,
 	nonconformity_type varchar(1024),
 	nonconformity_status_id bigint not null,
```

Comment:
Is this column still needed? This is a derived field now, right? So should we be dropping it here?

Reply:
The way the ticket was written and the checkboxes that existed when I started, I think I am supposed to create a second ticket that will permanently remove the non-conformity status "stuff" from the DB.